container/docker/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
@rossja is this a carry over from your repo or do you require we keep this license in place for the container? Ideally we'd just have the top level license for the whole repo, under NCC.